ui/src/literature/containers/CurateReferenceDrawerContainer.tsx

```diff
@@ -17,21 +18,29 @@ const stateToProps = (state: RootStateOrAny, { namespace }: { namespace: string
 
 const dispatchToProps = (dispatch: ActionCreator<Action>) => ({
   onDrawerClose() {
-    dispatch(setCurateDrawerVisibility(null));
+    dispatch(setCurateDrawerVisibility(null))
   },
 
   onCurate({
-    recordId,
+    id,
+    recordUuid,
     revisionId,
     referenceId,
     newReferenceId,
   }: {
-    recordId: number;
+    id: number;
+    recordUuid: number;
     revisionId: number;
     referenceId: number;
     newReferenceId: number;
   }) {
-    dispatch(curateReference({recordId, revisionId, referenceId, newReferenceId}));
+    dispatch(curateReference({id, recordUuid, revisionId, referenceId, newReferenceId}));
+  },
+
+  fetchSearchResults(namespace: string) {
+    dispatch(searchBaseQueriesUpdate(namespace, {
+      baseQuery: { q: '' },
+    }));
```

Comment:
I think here we can go back to the initial state of the drawer

Reply:
actually, with `destroyOnClose` do we even need it?

Reply:
It's to prepopulate the drawer with literature search results on initial enter, without the additional step of having to click search on empty search input. I thought it would be better this way, so the user sees all literature search results once the drawer opens, and the searchbar will only help to narrow the results.

Reply:
In all other drawers we have it like that, AFAIR. Could we please double check, if that's the case let's keep it consistent

Reply:
sure 👍